resources/pathway2Reactome_mappings.tsv

```diff
@@ -0,0 +1,15 @@
+pathway	target	reactomeId	description
+Androgen	AR	R-HSA-8940973	RUNX2 regulates osteoblast differentiation
+EGFR	EGFR	R-HSA-8856828	Clathrin-mediated endocytosis
+Estrogen	ESR1	R-HSA-8939902	Regulation of RUNX2 expression and activity
+Hypoxia	HIF1A	R-HSA-123456	XXX Pathway desc to be updated
+JAK.STAT	JAK1, JAK2, STAT1, STAT2	R-HSA-6785807	Interleukin-4 and 13 signaling
+MAPK	PRKMK1, MAP2K2, RAF1	R-HSA-2559580	Oxidative Stress Induced Senescence
+NFkB	NFKB1, TLR4, RELA	R-HSA-9020702	Interleukin-1 signaling
+PI3K	PIK3CA, PIK3CA	R-HSA-8853659	RET signaling
+TGFb	TGFBR1, TGFBR2	R-HSA-2173788	Down regulation of TGF-beta receptor signaling
+TNFa	TNFRSF1A	R-HSA-5357956	TNFR1-induced NFkappaB signaling pathway
+Trail	TNFSF10, BCL2, BCL2L1, BCL2L2, MCL1	R-HSA-123456	XXX Pathway desc to be updated

```

Comment:
Same as above, can the description be updated? R-HSA-123456 seems to be a placeholder and I couldn't easily find an alternative match. If not, should it be removed?

Reply:
This sort of errors were previously described in #713 and have yet to be addressed. I'm making a comment in the PR for clarification.